SPECS/kubernetes/kubernetes-1.17.spec

```diff
@@ -0,0 +1,207 @@
+%global debug_package %{nil}
+%ifarch x86_64
+%define archname amd64
+%endif
+%ifarch aarch64
+%define archname arm64
+%endif
+%define kubeproxy_n_1 1.16.13
```

Comment:
Curious, what do we need the kubeproxy_n_1 and coredns_n_1 for?

Reply:
I just left what was in the 1.17 spec file in msk8s. I actually don't know if it is needed (does not seem used anywhere in the .spec). I'm OK to remove it.

Reply:
discussed off line with George and these are left over => I'll remove that